cosmicfishpie/LSSsurvey/spectro_obs.py

```diff
@@ -127,73 +126,31 @@ def __init__(
         )
 
         self.observables = cfg.obs
+        self.specs = deepcopy(cfg.specs)
```

Comment:
The IM class does still use cfg.specs lets keep that in mind when we make it uniform

Reply:
all the cfgs are resolved now in newest commit